apps/web/app/(app)/gardens/[chain]/[garden]/pool/[poolId]/proposals/[proposalId]/page.tsx

```diff
@@ -10,7 +10,11 @@ import {
   getProposalDataQuery,
 } from "#/subgraph/.graphclient";
 import { formatTokenAmount, calculatePercentageBigInt } from "@/utils/numbers";
+import Image from "next/image";
 import { getIpfsMetadata } from "@/utils/ipfsUtils";
+import { UserIcon, InformationCircleIcon } from "@heroicons/react/24/outline";
+import { proposalStatus, poolTypes } from "@/types";
+import { proposalImg } from "@/assets";
```

Comment:
Looks like this image is too much pixelated, maybe we should implement the proposal identicon very soon, I will add an issue for that I the board:
https://github.com/orgs/1Hive/projects/12/views/1?pane=issue&itemId=69082828

Reply:
Agreed yesss ... I propose to add this issue in #257 where we are gonna upgrade the `Proposal` component and by consequence we can update the proposal page also